src/components/Settings/Export/Tabs/GeneralTabPanel.tsx

```diff
@@ -1,65 +1,78 @@
-import { CustomFormat, ExportApiFormatKey, IDocsEntity, useGetExportCitation } from '@api';
+import { CustomFormat, ExportApiFormatKey, IDocsEntity, useGetExportCitation, useGetUserSettings } from '@api';
 import { Stack } from '@chakra-ui/react';
 import { SampleTextArea } from '@components';
 import { ExportFormat, exportFormats } from '@components/CitationExporter';
-import { JournalFormatMap } from '@components/Settings/model';
+import { DEFAULT_USER_DATA, JournalFormatMap } from '@components/Settings/model';
 import { UserDataSetterEvent } from '@pages/user/settings/export';
 import { values } from 'ramda';
 import { Dispatch, useMemo } from 'react';
 import { ExportFormatSelect } from '../ExportFormatSelect';
 import { CustomFormatSelect } from '../CustomFormatSelect';
-import { useSettings } from '@lib/useSettings';
 
 export interface IGeneralTabPanelProps {
   sampleBib: IDocsEntity['bibcode'];
   dispatch: Dispatch<UserDataSetterEvent>;
   selectedOption: ExportFormat;
 }
 
-const exportFormatOptions = values(exportFormats);
-
 export const GeneralTabPanel = ({ sampleBib, selectedOption, dispatch }: IGeneralTabPanelProps) => {
-  const { settings: userSettings } = useSettings();
+  const { data } = useGetUserSettings({ initialData: DEFAULT_USER_DATA });
+
+  // fill any missing user data with default
+  const userSettings = useMemo(
+    () => (data ? { ...DEFAULT_USER_DATA, ...Object.entries(data).filter((s) => !!s) } : DEFAULT_USER_DATA),
+    [data],
+  );
+
+  const exportFormatOptions = values(exportFormats);
```

Comment:
This should already be happening inside of `useSettings`.  In there I'd update the `initialData` prop